singletonclient/SingletonClient/Implementation/SingletonUpdate.cs

```diff
@@ -222,9 +213,11 @@ private ILocaleMessages LoadOfflineLocaleBundle(string locale, string nearLocale
                 ).Split(',');
 
             List<string> componentList = _config.GetConfig().GetComponentList();
-            if (componentList == null)
+            if (componentList.Count == 0)
```

Comment:
What if componentList == null?

Reply:
GetComponentList() doesn't return null. it must return an object.

Reply:
sonarQube forced me to change from returning null to returning a list with no items.